templates/account/account.py

```diff
@@ -32,11 +35,28 @@ def ovc(self):
     def account(self):
         if not self._account:
             self._account = self.ovc.account_get(
-                                        name=self.name,
-                                        create=False
-                                     )
+                                        name=self.get_name(),
+                                        create=False)
```

Comment:
I'd use `name=self.data['name']` directly. `get_name` is more to use from external services.